privacy_evaluator/attacks/membership_inference/black_box.py

```diff
@@ -1,24 +1,24 @@
-from art.attacks.inference.membership_inference import MembershipInferenceBlackBox
-from typing import Tuple
 import numpy as np
 
-from privacy_evaluator.attacks.membership_inference.membership_inference import (
+from .membership_inference import (
     MembershipInferenceAttack,
 )
-from privacy_evaluator.classifiers.classifier import Classifier
+from ...classifiers.classifier import Classifier
 
 
 class MembershipInferenceBlackBoxAttack(MembershipInferenceAttack):
     """MembershipInferenceBlackBoxAttack class."""
 
+    _ART_MEMBERSHIP_INFERENCE_ATTACK_CLASS = "MembershipInferenceBlackBox"
+
     def __init__(
         self,
         target_model: Classifier,
         x_train: np.ndarray,
         y_train: np.ndarray,
         x_test: np.ndarray,
         y_test: np.ndarray,
-        attack_train_ratio: float = 0.5,
+        attack_model_type: str = "nn",
```

Comment:
Do you really want to hard-code that? Or do I misunderstand some internal logic here?

Reply:
It's a default value, so if `attack_model_type` is specified then whatever value is passed is checked to be either `rf`, `gb`, or `nn`. If it's not specified then `nn` is used.